Source/Core/VideoBackends/OGL/FramebufferManager.cpp

```diff
@@ -99,8 +99,8 @@ FramebufferManager::FramebufferManager(int targetWidth, int targetHeight, int ms
 
     glBindTexture(m_textureType, m_efbDepth);
     glTexParameteri(m_textureType, GL_TEXTURE_MAX_LEVEL, 0);
-    glTexImage3D(m_textureType, 0, GL_DEPTH_COMPONENT32F, m_targetWidth, m_targetHeight,
```

Comment:
Should we also avoid the depth-stencil format is bounding box is disabled?

`GL_DEPTH32F_STENCIL8` is defined along with `GL_DEPTH_COMPONENT32F` in ARB_depth_buffer_float, which is part of OpenGL 3.0 according to OpenGL Extensions Viewer, so it should work pretty much everywhere.